plugins/woocommerce-admin/client/products/fields/enriched-label.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Popover } from '@wordpress/components';
+import { Fragment, useState } from '@wordpress/element';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import { Icon, help } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './enriched-label.scss';
+
+export default function EnrichedLabel( {
+	helpDescription,
+	label,
+	moreUrl,
+	slug,
+} ) {
+	const [ isPopoverVisible, setIsPopoverVisible ] = useState( '' );
```

Comment:
Set the `useState( '' )` to `false` by default instead of an empty string.

Reply:
Good eyes! I fixed this in commit be3d7c6